postgres/datadog_checks/postgres/config.py

```diff
@@ -17,10 +17,26 @@ class PostgresConfig:
 
     def __init__(self, instance):
         self.host = instance.get('host', '')
+        if not self.host:
+            raise ConfigurationError('Please specify a Postgres host to connect to.')
         self.port = instance.get('port', '')
         if self.port != '':
             self.port = int(self.port)
+        self.user = instance.get('username', '')
+        if not self.user:
+            raise ConfigurationError('Please specify a user to connect to Postgres as.')
+        self.password = instance.get('password', '')
         self.dbname = instance.get('dbname', 'postgres')
+
+        self.application_name = instance.get('application_name', 'datadog-agent')
+        if PY3 and not self.application_name.isascii():
+            raise ConfigurationError("Application name can only include ASCII characters")
```

Comment:
```suggestion
            raise ConfigurationError("Application name can only include ASCII characters: %s", self.application_name)
```
a more informative error?

Reply:
also, move the word only to the right place: "...can include only ASCII..."